cluster/gce/configure-vm.sh

```diff
@@ -535,6 +535,17 @@ EOF
 node_labels: '$(echo "${NODE_LABELS}" | sed -e "s/'/''/g")'
 EOF
     fi
+    if [ -n "${ENABLE_NODE_AUTOSCALER:-}" ]; then
```

Comment:
I'd suggest to have both conditions under one `if` and only if ENABLE_NODE_AUTOSCALER = true.